test/fixtures/models/katello_repositories.yml

```diff
@@ -209,6 +208,17 @@ iso:
   gpg_key_id:           <%= ActiveRecord::Fixtures.identify(:fedora_gpg_key) %>
   content_view_version_id: <%= ActiveRecord::Fixtures.identify(:library_view_version_1) %>
 
+ostree_cdn:
```

Comment:
I feel like this ought to contain "rhel 7" in this as ostree_cdn is quite generic.